README.md

```diff
@@ -34,17 +34,22 @@ This repo contains scripts, instructions to manage your own selfie a day project
 ### Run
 
 * Clone/download this repo
+
 * Download [shape_predictor_68_face_landmarks.dat](https://sourceforge.net/projects/dclib/files/dlib/v18.10/shape_predictor_68_face_landmarks.dat.bz2) and place it in the cloned repo folder
-* Install dependencies :
-  ```
-  pip3 install wheel
```

Comment:
Did this because the installation of the other deps require this wheel

Reply:
Well ``wheel`` can also be installed via ``requirements.txt`` I suppose?